src/main/java/org/nixos/idea/psi/impl/AbstractNixPsiElement.java

```diff
@@ -5,7 +5,7 @@
 import org.jetbrains.annotations.NotNull;
 import org.nixos.idea.psi.NixPsiElement;
 
-abstract class AbstractNixPsiElement extends ASTWrapperPsiElement implements NixPsiElement {
+abstract public class AbstractNixPsiElement extends ASTWrapperPsiElement implements NixPsiElement {
```

Comment:
Because `AbstractNixString` needs to extend this class, and it has to be public (because Kotlin does not have package-private), and A cannot extend B if A is public and B is package-private

Writing `AbstractNixString` in Java would solve this

Reply:
Thanks. You don't have to change it. I was just wondering.